domain-cc/cc-api-controller/src/main/java/gov/va/vro/controller/cc/v3/ContentionClassificationController.java

```diff
@@ -0,0 +1,38 @@
+package gov.va.vro.controller.cc.v3;
+
+// import static gov.va.vro.model.redo.CamelConstants.POST_RESOURCE_QUEUE;
+// import static gov.va.vro.model.redo.CamelConstants.V3_EXCHANGE;
+
+import gov.va.vro.api.cc.ResourceException;
+import gov.va.vro.api.cc.v3.CCResource;
+import gov.va.vro.api.cc.v3.ResourceRequest;
+import gov.va.vro.api.cc.v3.ResourceResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class ContentionClassificationController implements CCResource {
+  @Override
+  public ResponseEntity<ResourceResponse> callEndpoint(String endpoint, ResourceRequest request)
+      throws ResourceException {
+    log.info("callEndpoint logging info");
+    try {
+      log.info("endpoint received: {}", endpoint);
+      log.info(endpoint);
+      log.info(endpoint);
+      log.info(endpoint);
```

Comment:
Another nitpick, maybe you don't need so many of these 😛

Reply:
won't fix this, this will be replace by rabbitmq call and I'm trying to keep PR's as small as possible